workflow-templates/prerelease.yml

```diff
@@ -53,14 +143,18 @@ jobs:
             echo "::error::'PROJECT' environment variable must be set"
             exit 1
           fi
+          if [ -z "${SYSTEM_NAME}" ]; then
+            echo "::error::'NAME' environment variable must be set"
+            exit 1
+          fi
           if [ -z "${DOMAIN}" ]; then
             echo "::error::'DOMAIN' environment variable must be set"
             exit 1
           fi
 
       - name: Set Environment URL
         run: |
-          echo "STG_ENV=https://${{ github.event.inputs.key }}.${DOMAIN}" >> $GITHUB_ENV
+          echo "STG_ENV=https://${SYSTEM_NAME}-${{ needs.prerelease.outputs.key }}.${DOMAIN}" >> $GITHUB_ENV
```

Comment:
@SatrioAjieWijaya @billyjulius review this please

Reply:
karena env var SYSTEM_NAME digunakan di job deploy dan regression, mungkin ada baiknya dijadikan sebagai workflow level environment variable 

atau mungkin variable STG_ENV di isi env variable url yang terdapat pada  job deploy 

gimana ?

Reply:
ta buat SYSTEM_NAME jadi workflow level env variable aja ya? soalnya ada ambil dari output key nya si prelease untuk subdomain yang digunakan

Reply:
@SatrioAjieWijaya apa ada concern lagi? klo engga bisa kita merge dulu untuk dicoba